### ISSUE TITLE
An Issue While Reporting Issues via VS Code

### ISSUE THREAD

**Issue by byyxx128** [microsoft/vscode#73921]
@rebornix I need your help.
I first noticed this issue long time ago, but I just confirmed it should be a terrible problem.

There are some issues written in Chinese with only one sentence:
> 所需的数据太大，无法直接发送。我们已经将其写入剪贴板，请粘贴。
(We have written the needed data into your clipboard because it was too large to send. Please paste.)

Initially I thought it was an prompt by VS Code. Soon afterwards there were so many issues with exactly the same contents. So I infer the sentence was written by VS Code rather than the authors.

Here are some issues I found:
#49252 #56451 #56452 #56449 #58028 #58052 #57746 #58478 #61309 #62998 #62434 #63636 #65470 #65184 #65851 #66835 #69765 #70024 #70635 #71331 #72739 #73026 #73706 #73868 #73935

Most of the issues are performance issues, which is the reason why the data was too large to send. Obviously most users didn't know what it meant. And dramatically, most maintainers did not understand as well. 😂 They asked the authors to report again in English. But is it useful to report the sentence in English? 🤔

In conclusion, I recommend to improve the way of showing the issue. Otherwise, many people cannot understand what they need to do.

**Comment by RMacfarlane** [comment]
@byyxx128 I have added a prompt that shows up in VS Code when reporting an issue when the data is too large to send:
![Screen Shot 2019-05-31 at 2 12 55 PM](https://user-images.githubusercontent.com/3672607/58735126-4e235f80-83ae-11e9-82b1-31c7eedb4c02.png)

This should get translated into other languages. I hope this will make it clearer to users that they will need to paste the data when they get to GitHub. Let me know if you have any other ideas on how to improve this!

**Comment by rebornix** [comment]
As discussed offline, we were wondering that if it's because that some users have many extensions (more than 50), which makes the URL exceeds the length of limit. One improvement we can do is filtering out inactive extensions instead of all enabled extensions.

**Comment by byyxx128** [comment]
Thanks all of you. Good ideas.

**Comment by rebornix** [comment]
It just came to my mind that the translation is somewhat misleading `所需的数据太大，无法直接发送`, users might interpret it as `VS Code needs a good amount of data and fails to send directly`. The new notification mentions `GitHub`, which can probably rule this out.

**Comment by byyxx128** [comment]
Exactly, the translation is obviously a little bit confusing that definitely caused the users misunderstanding. And the original English description is not enough clear as well. Another serious reason is that the users may just click the ![image](https://user-images.githubusercontent.com/20079289/61516965-1f9c2b00-a9d4-11e9-99e6-756123499bca.png) button directly without reading the contents in the text box. Especially for the new users of GitHub and the ones who are not good at English, they thought they did a good job successfully, but dramatically the fact is opposite.

**Comment by byyxx128** [comment]
![image](https://user-images.githubusercontent.com/20079289/61518993-d7cbd280-a9d8-11e9-90cb-09954fc19dc0.png)
As shown above, there is a close button on the dialog on Windows. However, it is same either to click the `Yes` button or the `×` button. I think the function of `×` button should be same as the `Cancel` button rather than the `Yes` button, or the `×` button can be disabled.

**Comment by RMacfarlane** [comment]
Closing this as does-not-reproduce, as the current behavior is that the `x` acts as `Cancel`
